lib/inspec/runner.rb

```diff
@@ -201,9 +200,9 @@ def add_target(target, _opts = [])
 
     def supports_profile?(profile)
       unless profile.supports_runtime?
-        raise "This profile requires #{Inspec::Dist::PRODUCT_NAME} version "\
+        raise "This profile requires the inspec-core gem version "\
```

Comment:
InSpec should be allowed to refer to itself by product name. It would be better to tell the user that they need "Chef InSpec" or the community package at a particular version; as that's the item they will need to download. Referring to the gem might mislead the user that they need to adjust gems in their current product installation.

More generally Chef InSpec should be allowed to refer to itself anywhere in its code. It shouldn't need to "unbrand". Is it possible for the community distribution to manage a modified version of `dist.rb` so that `Inspec::Dist::PRODUCT_NAME` and the other dist consts refer to the community distribution names?

I'm available on the community slack if I'm missing some of the context here and you'd like to discuss

Reply:
Lemme try to reformulate the problem: I use kitchen-terraform, which uses the Inspec lib under the hood. That line i'm changing is the _only_ direct reference to Chef Inspec in the library proper.

 Removing it means tools like kitchen-terraform are trademark compliant. Leaving it in puts a lot of the community in a really weird place; Do projects like Cinc need to redistribute kitchen-terraform to make it compliant? Or does the burden fall on the authors to publish and use new inspec-core gems? Should we have 2 versions of every such product, one with Inspec and one with a modified lib?

As an alternative to this PR, perhaps Chef Legal could issue a statement here or somewhere else official stating that this particular occurence of "Chef Inspec" is cool with them. Something I can reference when folks raise this particular issue.

What's your handle on Slack? We can pick this up over there this week sometime if this still doesn't convey the perceived issue correctly.

Reply:
Hi @bobchaos -

I'm `Stocksy` in the Chef community Slack. We can ask Chef Legal about this , but I wanted to clarify:
Does Cinc include kitchen-terraform in its distribution? Or do Cinc users download and install kitchen-terraform separately from Cinc?
I believe you mean the latter; but we wanted to confirm before asking Legal.

Thanks

Reply:
It is indeed the latter scenario at this time; we're asking because, as the Cinc Project, we want to know if we should align on the former in the future (or find someone willing to do that, whether or not kitchen-terraform is in the project's scope is up for debate)

You could say I'm asking more as a service to the community than as a directly interested party

Reply:
Hi @bobchaos - 

Sorry for slow response.

I asked Chef Legal about Chef trademarks appearing in CINC logging due to kitchen-terraform being added to the CINC distribution. I got this statement:

> CINC (or any distribution) is permitted to include/reference 3rd party components that ultimately depend on Chef's original components. This is not a trademark violation, just a side effect of a highly connected OSS ecosystem.

I hope this statement is useful.

Regarding this PR specifically, I think it's a good idea to remove hardcoded distribution details from inspec-core, but they should still be accessible to the runner class. Clinton made a good point that when the InSpec run is in the middle of your full automation stack, it's super unhelpful for error messages to not hint at the component that raised the error. And Chef InSpec should be able to continue to refer to itself as Chef InSpec.

Reply:
This 100% resolves the issue as far as I'm concerned, thanks! I'ma go ahead and close this.